### ISSUE TITLE
Diagonal scroll

### ISSUE THREAD

**Issue by murata2makoto** [edrlab/bd-comics-manga#11]
Is diagonal scroll (as demonstrated by [this page](https://codepen.io/kaltoft/full/MoVmVR)) in the scope?

**Comment by anymus** [comment]
This kind of diagonal scroll is not something we have considered for the first version of the standard - only purely horizontal and vertical scrolls are taken into account at this stage, which means that both (a) the user gesture and (b) the corresponding "camera movement" (supposing a camera in a larger 2D workspace pans over a set of images spread over a flat surface) are either horizontal or vertical, along the same axis, and in the same direction.

I had a hard time figuring out how the example you are pointing to is supposed to work on the user side: the "camera movement" is actually controlled by a vertical scroll triggered within the diagonal slider's area (if you start scrolling outside of that area, then a normal vertical page scroll occurs). Also, when rou reach the slider's end (when scrolling down) or beginning (when scrolling up), scrolling defaults back to a normal vertical page scroll. I would not describe that as highly intuitive UX-wise. 

We talked about the possibility to allow other kinds of "camera paths" in future versions, which would allow for a disconnect between user gesture and camera movement (e.g. the user scrolls right but the camera actually moves down, or follows a complex trajectory going up and down), however the example you are pointing to adds one more level of complexity: the scrollbar and slider area in general (the overall shape) are diagonal.

**Comment by murata2makoto** [comment]
I agree that the UI of this example is not very intuitive because of the mixture of a normal vertical page scroll and diagonal scroll.  I would use diagonal scroll only.

I referenced this example only to demonstrate that diagonal scroll is already possible without using Javascript.

**Comment by anymus** [comment]
It is a clever use of HTML and CSS indeed! Thanks for pointing it out.
